<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudyPower</class>
 <widget class="QDialog" name="StudyPower">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>471</width>
     <height>691</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="main_widget">
    <widget class="QLabel" name="main_username">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>user :</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_rank_num">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>rank :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="main_ranking_list">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>360</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>排行榜</string>
     </property>
    </widget>
    <widget class="QPushButton" name="main_train">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>430</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>个人训练赛</string>
     </property>
    </widget>
    <widget class="QPushButton" name="main_rank">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>500</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>排位赛</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="train_widget">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>391</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>个人训练模式,共有五道题</string>
     </property>
    </widget>
    <widget class="QPushButton" name="train_suiji">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>330</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>随机题库</string>
     </property>
    </widget>
    <widget class="QPushButton" name="train_lishi">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>390</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>历史题库</string>
     </property>
    </widget>
    <widget class="QPushButton" name="train_changshi">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>450</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>常识题库</string>
     </property>
    </widget>
    <widget class="QPushButton" name="train_gushi">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>510</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>古诗题库</string>
     </property>
    </widget>
    <widget class="QPushButton" name="train_back">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>590</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="train_run">
    <widget class="QLabel" name="train_question">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>401</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>题目</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLCDNumber" name="train_lcdNumber">
     <property name="geometry">
      <rect>
       <x>313</x>
       <y>20</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="train_right_answer">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>630</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>401</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="train_A">
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="train_B">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="train_C">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="train_D">
        <property name="text">
         <string>D</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="train_result">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>241</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>     训练结束</string>
     </property>
    </widget>
    <widget class="QLabel" name="train_score">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>381</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>训练结果</string>
     </property>
    </widget>
    <widget class="QPushButton" name="train_result_back">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>600</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="rank_wait">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>301</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>等待匹配中. . .</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="rank_match_result">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>匹配成功</string>
     </property>
    </widget>
    <widget class="QLabel" name="rival_name">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>rival name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="rival_rank">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>rival rank:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="rank_run">
    <widget class="QLCDNumber" name="rank_lcdNumber">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>401</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="rank_A">
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rank_B">
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rank_C">
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rank_D">
        <property name="text">
         <string>D</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="rank_question">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>411</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>题目</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="rank_current_score">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Score:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="rank_result">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>排位结束</string>
     </property>
    </widget>
    <widget class="QLabel" name="rank_my_score">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>我方得分:</string>
     </property>
    </widget>
    <widget class="QLabel" name="rank_rival_score">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>310</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>对方得分:</string>
     </property>
    </widget>
    <widget class="QLabel" name="rank_my_result">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>390</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>排位结果:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>590</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="rivallist">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>28</pointsize>
      </font>
     </property>
     <property name="text">
      <string>排行榜</string>
     </property>
    </widget>
    <widget class="QLabel" name="rival_first">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>441</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="rival_second">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>461</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="rival_third">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>461</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>560</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Adobe 黑体 Std R</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
